docs/schema/view-element-module.md

```diff
@@ -177,7 +178,7 @@ The following sections describe attributes, child elements, and parent elements.
 <li><p>**Recursive**—Show all files of all folders.</p></li>
 <li><p>**RecursiveAll**—Show all files and all subfolders of all folders.</p></li>
 </ul>
-<p>If this attribute is not set, the view displays only the files and subfolders of a specific folder. The value of this attribute corresponds to the <span sdata="cer" target="P:Microsoft.SharePoint.SPView.Scope"><span class="nolink">Scope</span></span> property of the <span sdata="cer" target="T:Microsoft.SharePoint.SPView"><span class="nolink">SPView</span></span> class.</p></td>
+<p>If this attribute is not set, the view displays only the files and subfolders of a specific folder. The value of this attribute corresponds to the <span><span>Scope</span></span> property of the <span><span>SPView</span></span> class.</p></td>
```

Comment:
We need to turn all HTML tables we touch into Markdown.  I sent you a Markdown table for this one, since it's very long.